packages/lexical-devtools/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link
+      rel="icon"
+      type="image/svg+xml"
+      href="/src/images/lexical-white.png" />
+    <!-- <link rel="manifest" href="/src/manifest.json" /> -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Lexical Developer Tools</title>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script type="module" src="/src/main.jsx"></script>
```

Comment:
You can import `.tsx` files directly and it will update them to `.js`.

```suggestion
    <script type="module" src="/src/main.tsx"></script>
```